information/licensing/Concurrent_sessions_based_licensing.md

```diff
@@ -0,0 +1,66 @@
+# Concurrent Sessions-Based Licensing
```

Comment:
Change the file name to be in kebab-case and not snake_case.

Reply:
Regarding remarks: 

The note for the servers is unnecessary. Just specify the user actions. - The note for the servers is removed. Used "@@name (ERP.net) instance" instead.

Reply:
Regarding remarks: 

Change the file name to be in kebab-case and not snake_case. - The file was named in kebab-case style, in accordance with other files in this directory.